app/models/reservation.rb

```diff
@@ -321,6 +317,23 @@ def update(current_user, new_params, new_notes) # rubocop:disable all
     end
   end
 
+  def add_notes(current_user, contents)
+    return false if contents.empty?
```

Comment:
Also, this doesn't work... I manually just clicked the "Add Notes" button on a reservation with nothing in it and it still worked...